api/envoy/service/ext_proc/v3/external_processor.proto

```diff
@@ -195,7 +195,7 @@ message ProcessingResponse {
 message HttpHeaders {
   // The HTTP request headers. All header keys will be
   // lower-cased, because HTTP header keys are case-insensitive.
-  config.core.v3.HeaderMap headers = 1;
+  config.core.v3.HeaderMapBytes headers = 1;
```

Comment:
So I am wondering have we decided how to deal with breaking API change, if they are many clients out there?

At least I think we need release note here.

Reply:
Yes, a runtime flag is added to protect the original behavior.  However, it might be ideal set the run-time flag to a value to make breaking change as default to get the ext_proc filter to the state which by default support non-utf8 characters.  Let's see what other people think.